crossplane.yaml

```diff
@@ -12,7 +12,8 @@ metadata:
       This configuration serves as a reference platform for cloud native services
       and applications running in AWS.
 spec:
-  crossplane: v0.13.0
+  crossplane:
+    version: ">=v0.14.0-0"
   dependsOn:
-  - provider: crossplane/provider-aws
-    version: v0.12.0
+    - provider: crossplane/provider-aws
+      version: v0.13.0
```

Comment:
Just a note: when this field is active, we will support version constraints in the same manner as the Crossplane version. This would be valid (exact version), but you could also do a range as you did above 🙂

Reply:
ah, that's good to know, thanks @hasheddan!